Sources/Extensions/TableDirector/TableRow+Extensions.swift

```diff
@@ -34,3 +34,12 @@ public extension TableRow where CellType.T: SeparatorCellViewModel {
     }
 
 }
+
+public extension TableRow where CellType: SeparatorCell, CellType.T: SeparatorCellViewModel {
+
+    /// TableRow typed as SeparatorRowBox
+    public var separatorRowBox: SeparatorRowBox {
```

Comment:
`public` is redundant. May be rename to `rowBox`?

Reply:
I think "rowBox" is not very clear name, because it does not contain "separator"
May be separatorBox?